torch/quantization/observer.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, dtype):
         self.dtype = dtype
 
     @abstractmethod
-    def forward(self, x):
-        pass
+    def forward(self, x: torch.Tensor):
```

Comment:
Curious as to why this change is needed for scriptability. Since we override forward, shouldnt that be sufficient?

Reply:
reverted